2007-10-28  Aurelien Jarno  <aurelien@aurel32.net>

        * linuxthreads/sysdeps/pthread/bits/libc-lock.h (__libc_once_get): New
        definition.

---
 linuxthreads/sysdeps/pthread/bits/libc-lock.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/linuxthreads/sysdeps/pthread/bits/libc-lock.h
+++ b/linuxthreads/sysdeps/pthread/bits/libc-lock.h
@@ -242,6 +242,9 @@
     }									      \
   } while (0)
 
+/* Get once control variable.  */
+#define __libc_once_get(ONCE_CONTROL) \
+  ((ONCE_CONTROL) != PTHREAD_ONCE_INIT)
 
 /* Start critical region with cleanup.  */
 #define __libc_cleanup_region_start(DOIT, FCT, ARG) \
